README.md

```diff
@@ -99,16 +97,77 @@ end
 ## Specifying a default Facter version
 
 By default, `on_supported_os` will return the facts for the version of Facter
-that it has loaded (usually this is Facter 2.5.1). This behaviour can be
-overridden by setting the `default_facter_version` RSpec setting in your
-`spec/spec_helper.rb` file.
+that it has loaded. It will check for the loaded Puppet version and maps that to
+the Facter version that Perforce released in the matching AIO. The mapping is
```

Comment:
Is this clearer?

```suggestion
that it has loaded.
It will match the Facter version based on the loaded Puppet version to what the official packages provide.
The mapping is
```